### PR TITLE
LPS-161922 | Automate Test | Master

### PR DESCRIPTION
**Pull request by JhossephG**
https://issues.liferay.com/browse/LPS-161922

### CODE REVIEW

**Comment by JhossephG** [comment]
ci:test:sf

**Comment by JhossephG** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase [hunk 1]
```diff
@@ -83,6 +83,86 @@ definition {
...
+		}
+
+		task ("Go to the Queue tab") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Notifications",
+				panel = "Control Panel",
+				portlet = "Queue");
+		}
+
+		task ("Then delete the notification and assert assert the notification is no longer showed") {
```
Comment (evanilson-santana) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase:157]: Typo: just one "assert"

**Comment by evanilson-santana** [comment]
@JhossephG I made cherry-pick for your commits and send to the new PR: [82](https://github.com/liferay-bpm-qa/liferay-portal/pull/82)

**Pull request closed without merging**